Tasks/RunDistributedTests/task.json

```diff
@@ -245,9 +245,8 @@
      },
      {
          "target": "testSuite",
-         "endpoint":"/$(system.teamProject)/_apis/test/plans/$(testPlan)/suites",
-         "selector":"jsonpath:$.value[*].name",
-         "keySelector":"jsonpath:$.value[*].id",
```

Comment:
This is removed only for MultiSelectTree control. This functionality is needed for two other controls and hence cannot be removed!